### ISSUE TITLE
Build out the information, contact numbers screen

### ISSUE THREAD

**Issue by lukesleeman** [Western-Health-Covid19-Collaboration/wh_covid19_app#44]
Inside the information screen there is a button which takes you to the contact numbers screen.  That screen is currently **really** basic / possibly doesn't exist.

**We need to build build out a detailed version of the contact numbers screen**

@greggmiller Can you please provide some information in this issue on what should be in this section?  Once that is done assign the issue to Marc Edwards.

@marcedwards Once greg has provided info, can you provide some design guidance in this issue in on how it should look.  Once this is done, can you label this issue as "Ready for Dev" so that one of the devs can pick it up.

From a code POV the phone numbers in the app will be referenced from multiple places.  So it's best to define them as constants in one place in the codebase.

**Comment by lukesleeman** [comment]
Based off what @Mattro told me, there will be numbers sections in this screen:
> eg: Look after yourself. Tap call, get a help line.
> eg2: Using a ventilator. tap call, get ventilator support.

Then we will do something like
> Tap -> Confirmation -> Call
or even
> Tap -> Options -> Tap -> Confirmation -> call

These same numbers will be accessed from other sections in the app.  We will track that work via another issue.

**Comment by greggmiller** [comment]
Its the numbers from this page we need: 
http://wh.cyphix.net/

These are the main numbers from across the hospital.
However in first instance a link to this page which opens it up might suffice?

**Comment by greggmiller** [comment]
I will in due course try and compile a list of numbers fit for purposes, but this probably isnt a priority for first release.

**Comment by lukesleeman** [comment]
Yeah, I like just having the button link to the page.  That way we know it's up to date.  We can do proper layout and other bits for this a few days down the track, once we have filled out more of the app.

Luke

**Comment by lukesleeman** [comment]
I'm tagging this as ready for dev.  Can one of the @Western-Health-Covid19-Collaboration/flutter-devs link up the button to launch http://wh.cyphix.net/ in a web browser?

We will design a proper page a bit down the track.

**Comment by alessandrofavero** [comment]
I can pick this issue tonight after work

**Comment by alessandrofavero** [comment]
Implemented with PR #58

**Comment by lukesleeman** [comment]
@alessandrofavero I can see the code in there for the contact numbers, but when I run the latest firebase build (which should have this) tapping the contact numbers in the information screen does nothing.

I'm re-opening this.  If I'm somehow getting this wrong and it is actually working, just close it again

**Comment by alessandrofavero** [comment]
Hey @lukesleeman, my bad I didn't connect that card I will do it now. Can you please confirm that by going:
home page -> your welfare --> click on the action button on the navbar
takes you to the web page? Thanks

**Comment by lukesleeman** [comment]
Ahhhhh, I see.  It's good that the phone icon in that screen takes you there.  But also clicking the little info icon on the home screen, then going to contact numbers should take you to the same web page

Luke
